package.json

```diff
@@ -32,7 +32,7 @@
     "svg4everybody": "^2.1.9"
   },
   "devDependencies": {
-    "@justeat/gulp-build-fozzie": "^7.13.0",
+    "@justeat/gulp-build-fozzie": "7.17.0",
```

Comment:
Some inconsistency here with `^`

Reply:
Any new packages should be specified without the `^` going forward. I held off updating the others as I wanted to make sure these changes worked correctly. 

I'll go and update & test the other now.

Reply:
What's the reason for this?

Thought this gets pinned as part of the lock file anyway doesn't it?

Reply:
Yeah they do get pinned which means that the `^` is kind of redundant anyway. There is a good description of why pinning is recommended [towards the end of this post](https://renovateapp.com/docs/deep-dives/dependency-pinning), but basically it makes it clearer to see exactly which dependencies are in use by looking at the package.json file rather than the lock file. 

Which upgrade cycle are you referrring to?

Reply:
I just checked the upgrades - it doesn't affect that which is good 😄 

I'm not sure I really understand why doing this is beneficial on top of a lockfile though - if we weren't using a lockfile it would make complete sense, but it seems like we'd also need to add a `yarn.rc` file to every repo too so that new installs didn't have ^ carets applied, else we'd have to always delete those that get applied to new packages.

Not sure if I'm missing something though 🔍  If we think this is worthwhile, we should be ensuring all future packages added also don't have carets added to their version numbers to keep consistency.

Ultimately, I don't really mind if the `package.json` modules have carets or not - just don't want to give ourselves more work across every existing repo unless we think it's valuable to do so 🤔

Reply:
I'm happy to keep using carets if that's what everyone wants. I just prefer to be able to look at the package.json and see the exact dependencies which are in use. With the carets the version isn't neccesarily what is in use i.e. package.json says v3.2.1 but the lockfile says v3.2.11 which means that the package.json version is a little misleading.

We shouldn't have to add a yarnrc file, we're not doing anything funky just explicitly stating which versions we want to use.

Also, we don't neccessary have to update all of the modules at once in this way, we could do it as we make other changes.

Reply:
Right, I see yeah, that could become annoying without the config file. I'll create an issue and get everyones thoughts on this. I'll revert the changes here for now until we've made a decision.